package.json

```diff
@@ -6,6 +6,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "build": "kaliber-pack"
```

Comment:
Check even hoe dit bij de cache library is ingesteld, volgens mij bouwt hij daar automatische voor dat je een release doet naar npm